docs/07. media library/01. configuration.md

```diff
@@ -0,0 +1,9 @@
+# Configuration
+
+The media library uses the [Liip Imagine Bundle](http://symfony.com/doc/1.0/bundles/LiipImagineBundle/index.html) to serve the images.
+
+The media library in fork comes with 5 predefined filters. For more information about this you can check the [official documentation](http://symfony.com/doc/1.0/bundles/LiipImagineBundle/filters.html).
```

Comment:
Suggestions to make things specific:
- "For more information about **how to configure filters** you can ..."
- [official **Liip Imagine Bundle** documentation]